cns/NetworkContainerContract.go

```diff
@@ -95,6 +96,8 @@ type CreateNetworkContainerRequest struct {
 	AllowHostToNCCommunication bool
 	AllowNCToHostCommunication bool
 	EndpointPolicies           []NetworkContainerRequestPolicies
+	BackendNetworkInterfaceID  string
```

Comment:
can it move inside a structure which contains backendnetwork info?

Reply:
yes I can add a struct `BackendNetworkInfo` with these 2 new fields

Reply:
added

Reply:
@tamilmani1989  - should I be reusing the fields added by PR - https://github.com/Azure/azure-container-networking/pull/2250/commits/3b1c720a68c37c4783305dece4bfcbd873a29d23#diff-cc7993ab0a3ea963068707fda61d4664fbac1ccf3ab3bfaf0949491d90ad7f3fR85 - only thing I'm not sure of is why PodIpInfo is being used instead of adding this to `CreateNetworkContainerRequest`, how are these fields sent to CNS by DNC & to be consumed by DNC? I was making this change so DNC can create a NCRequest with these fields to send to CNS..not sure if that part of change has been implemented yet for swiftv2? @rbtr